phira/locales/en-US/settings.ftl

```diff
@@ -3,61 +3,64 @@ label = SETTINGS
 
 general = General
 audio = Audio
-chart = Beatmap
+chart = Chart
 debug = Debug
 about = Info
 
 item-lang = Language
-item-offline = Offline mode
-item-offline-sub = You can't upload playing record in offline mode
-item-server-status = Server status
-item-server-status-sub = Navigate to the webpage to check the server status
-check-status = Check
+item-offline = Offline Mode
+item-offline-sub = Disables all online functionality.
+item-server-status = Server Status
+item-server-status-sub = Opens the server status page in your browser.
+check-status = Open
 item-mp = Multiplayer
-item-mp-sub = Enable multiplayer mode
-item-mp-addr = Multiplayer server
-item-mp-addr-sub = Server address, 'host:port'
-item-mp-addr-invalid = Invalid server address
-item-lowq = Low quality mode
-item-lowq-sub = Enable this if the UI is laggy
-item-insecure = Insecure mode
-item-insecure-sub = Enable this if you can't use online functionalities. Makes your connection insecure!
+item-mp-sub = Enables multiplayer functionality.
+item-mp-addr = Multiplayer Server
+item-mp-addr-sub = Allows you to connect to a custom multiplayer server.
+item-mp-addr-invalid = Invalid server address.
+item-lowq = Low Resolution Mode
+item-lowq-sub = Lowers the quality of the UI.
+item-insecure = Insecure Connection
+item-insecure-sub = Enables old devices to use online functionality.
 item-enable-anys = Enable Anys
-item-enable-anys-sub = Utilize Anys gateway to enhance connection quality
-item-anys-gateway = Anys gateway
-item-anys-gateway-sub = Anys gateway address
-item-anys-gateway-invalid = Invalid gateway address
-
-item-adjust = Automatic time adjustment
-item-adjust-sub = Adjusts time dynamically to sync the music and beatmap
-item-music = Music volume
-item-sfx = Sound effect volume
-item-bgm = BGM volume
-item-cali = Adjust offset
-
-item-show-acc = Show real-time accuracy
-item-dc-pause = Double tap to pause
-item-dhint = Highlight simul. notes
-item-dhint-sub = Notes that touch line simultaneously will be highlighted
-item-opt = Aggressive optimization
-item-opt-sub = Improves performance, but can cause incorrect behavior
+item-enable-anys-sub = Allows you to use an Anys gateway to improve network stability.
+item-anys-gateway = Anys Gateway
+item-anys-gateway-sub = Allows you to use a custom Anys gateway address.
+item-anys-gateway-invalid = Invalid gateway address.
+
+item-adjust = Automatic Time Adjustment
+item-adjust-sub = Adjusts the audio and chart offset dynamically.
+item-music = Music Volume
+item-sfx = SFX Volume
+item-bgm = BGM Volume
+item-cali = Adjust Offset
+
+item-show-acc = Real-Time Accuracy
+item-dc-pause = Pause Confirmation (2x)
```

Comment:
It's accept to pause with one tap, this description may make someone think I can only pause when I enable this option